src/lib/components/Accordion/AccordionItem.svelte

```diff
@@ -11,9 +17,28 @@
 
   let expanded: boolean;
   $: expanded = $expandedItems[id] ?? false;
+
+  // strip out everything except characters, digits,
+  // and whitespace, then convert to Kebab case.
+  // E.g. "Title 2: This time it's personal"
+  // becomes "title-2-this-time-its-personal"
+  // TODO: Make `titleHref` come from Contentful instead of deriving it here
+  const titleHref = title
+    ? title
+        .replace(/[^A-Za-z0-9\s]/g, "")
+        .toLowerCase()
+        .split(" ")
+        .join("-")
+    : null;
+
+  onMount(() => {
+    if ($page.url.hash === `#${titleHref}` && $expandedItems) {
+      $expandedItems[id] = true;
+    }
+  });
```

Comment:
Let's use `afterNavigate` instead of `onMount`. `afterNavigate` will run whenever `onMount` does, but _also_ when navigating to a new URL.